server/controllers/listings/by_element_id.ts

```diff
@@ -0,0 +1,25 @@
+import { property } from 'lodash-es'
+import { getElementById } from '#controllers/listings/lib/elements'
+import { getListingById } from '#controllers/listings/lib/listings'
+import { unauthorizedError } from '#lib/error/error'
+import { filterVisibleDocs } from '#lib/visibility/filter_visible_docs'
+import type { ListingElement } from '#types/element'
+
+const sanitization = {
+  id: {},
+}
+
+async function controller ({ id, req, reqUserId }) {
```

Comment:
It would be nice to have a couple of tests for this controller (in particular: what if we request a non-existing element, a non-authorized element). It would have revealed that `req` is not defined on the controller params object, but as second argument.

```suggestion
async function controller ({ id, reqUserId }, req) {
```

But we actually don't need to access it here, see below.

Reply:
I added some tests in 08fdfab86 which logically fixed the bugs you mentioned